documentation/modules/ova-prerequisites.adoc

```diff
@@ -19,17 +19,17 @@ Migration of OVA files that were not created by VMware vSphere but are compatibl
 
 ** In one or more compressed Open Virtualization Format (OVF) packages that hold all the VM information.
 +
-The filenames of the compressed packages *must* have `.ova` extensions. Several packages can be stored either in the same folder or in individual folders:
+The filenames of the compressed packages *must* have `.ova` extensions. Several packages can be stored either in the same folder or in individual folders. When this structure is used, {project-short} scans the root folder and the first-level subfolder
 +
-`<nfs_shared_directory>/folder1/vm1.ova` +
-`<nfs_shared_directory>/folder1/vm2.ova` (`vm1.ova` and `vm2.ova` in `folder1`)
-+
-_or_
-+
-`<nfs_shared_directory>/folder1/vm1.ova` +
-`<nfs_shared_directory>/folder2/vm2.ova` (`vm1.ova` in `folder1`, `vm2.ova` in `folder2`)
+For example, if the NFS share is, `/nfs`, then: +
+The folder `/nfs/vm.ova` is scanned. +
```

Comment:
but in this section we refer to compressed packages, i.e., tar files so /nfs/vm.ova is a file (not a folder)
```suggestion
The folder `/nfs` is scanned. +
```